tests/sentry/mediators/sentry_app_installations/test_destroyer.py

```diff
@@ -143,7 +143,11 @@ def test_deletes_on_request_exception(self):
 
     @responses.activate
     def test_fail_on_other_error(self):
+        from sentry.constants import SentryAppStatus
+
         install = self.install
+        self.sentry_app.update(status=SentryAppStatus.PUBLISHED)
```

Comment:
@ceorourke Do we need this still? We are only ignoring the error at the task level, there should still be an `IgnorableSentryAppError` raised